run-cassandra-centos.sh

```diff
@@ -0,0 +1,366 @@
+
+Example:
+      $0 -i centos-7 -k g4-general-16G -p us-east-3b
+
+Options:
+      -h Show this message
+      -i The image you want to test. Can be UUID or image name.
+      -k The package you want to use.  Can be UUID or package name.
+      -p The profile you wish to use with Triton CLI.
+
+This script will:
+      - Create 3 instances, using the specified image and package.
+      - NOTE: This test will work only for an lx image with CentOS.
+      - Install Cassandra software, and configure a 2-node Cassandra cluster.
+      - Install and configure the YCSB benchmark tool.
+      - Execute the YCSB benchmark tests against the Cassandra cluster.
+      - NOTE: This script assumes you have the Triton CLI command tool setup
+        with a valid profile for your test environment.
+
+EOF
+exit 1
+}
+
+while getopts “hi:k:p:” OPTION
+do
+  case $OPTION in
+    h)
+      usage
+      exit 1
+      ;;
+    i)
+      IMAGE=$OPTARG
+      ;;
+    k)
+      PACKAGE=$OPTARG
+      ;;
+    p)
+      PROFILE=$OPTARG
+      ;;
+    ?)
+      usage
+      exit
+      ;;
+  esac
+done
+
+if [[ -z $IMAGE ]] || [[ -z $PACKAGE ]] || [[ -z $PROFILE ]]; then
+    usage
+    exit 1
+fi
+
+#--------------------------------------------------------------------
+# Function definitions:
+#
+
+get_image_details() {
+    echo "Getting image details:"
+    IMAGEDETAILS=$(triton -p ${PROFILE} image get $1 | json -a name version id)
+    IMAGENAME=$(echo $IMAGEDETAILS | cut -d ' ' -f 1)
+    VERSION=$(echo $IMAGEDETAILS | cut -d ' ' -f 2)
+    UUID=$(echo $IMAGEDETAILS | cut -d ' ' -f 3)
+    echo "    $IMAGEDETAILS"
+    echo ""
+}
+
+get_package_details() {
+    echo "Getting package details:"
+    PACKAGEDETAILS=$(triton -p ${PROFILE} package get $1 | json -a name version id)
+    PACKAGENAME=$(echo $PACKAGEDETAILS | cut -d ' ' -f 1)
+    PACKAGEVERSION=$(echo $PACKAGEDETAILS | cut -d ' ' -f 2)
+    PACKAGEUUID=$(echo $PACKAGEDETAILS | cut -d ' ' -f 3)
+    echo "    $PACKAGEDETAILS"
+    echo ""
+}
+
+set_name() {
+    NAME="${IMAGENAME}-${VERSION}-${DATE}"
+    echo "Instance name prefix:"
+    echo "    ${NAME}"
+    echo ""
+}
+
+get_networks() {
+	echo "Getting networks:"
+    
+    PUBLIC_NETWORK=$(triton -p ${PROFILE} network list -j | json -ag id -c 'this.public === true' | head -1)
+	PRIVATE_NETWORK=$(triton -p ${PROFILE} network list -j | json -ag id -c 'this.public === false' -c 'this.fabric !== true' | head -1)
+    
+    # Trying using a fabric network instead
+    if [[ -z "$PRIVATE_NETWORK" ]]; then
+        PRIVATE_NETWORK=$(triton -p ${PROFILE} network list -j | json -ag id -c 'this.public === false' | head -1)
+    fi
+    
+    echo "    Public:  $PUBLIC_NETWORK"
+    echo "    Private: $PRIVATE_NETWORK"
+    echo ""
+}
+
+create_instance() {
+    echo "Creating instance ${1}:"
+    triton -p ${PROFILE} instance create -w -n $1 -N $PUBLIC_NETWORK -N $PRIVATE_NETWORK -t $TAG --script=$SCRIPT -M $METADATAFILE $2 $3
```

Comment:
Per my previous comment, I think this part and all the supporting code in the script can be removed:

```
--script=$SCRIPT -M $METADATAFILE
```